tools/gyp/pylib/gyp/generator/msvs.py

```diff
@@ -2662,6 +2662,15 @@ def _GetMSBuildGlobalProperties(spec, guid, gyp_file_name):
     else:
       properties[0].append(['ApplicationType', 'Windows Store'])
 
+  msvs_windows_sdk_version = None
+  if msvs_windows_sdk_version == None and version.ShortName() == '2017':
+    vs2017_sdk = '10.0.14393.0'
+    vs2017_sdk = os.environ.get('vs2017_sdk', vs2017_sdk)
+    if vs2017_sdk:
```

Comment:
Yeah, AFAIK, there is not way to set empty variable in `cmd`. Empty variable is equivalent of not defining environment variable. Python, definitely you could do that in which case the check makes sense.